src/Kodify/BlogBundle/Controller/PostsController.php

```diff
@@ -61,4 +91,27 @@ public function createAction(Request $request)
 
         return $this->render('KodifyBlogBundle:Default:create.html.twig', $parameters);
     }
+
+    /**
+     * Rates a post
+     *
+     * @param $id post
+     * @param $star number of stars
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function rateAction($id, $star)
+    {
+        if ($this->getDoctrine()->getRepository("KodifyBlogBundle:Post")->setRate($id, $star)) {
```

Comment:
Unfortunately I didn't choose the best name for that method, maybe only "Rate" would be better. My intention was to avoid the needing of Entity Object as a param to make an easier reusability on different places.